web-ui/enterprise-reporting/reporting_ui.markdown

```diff
@@ -13,12 +13,12 @@ If you are familiar with SQL syntax, you can input your query into the interface
 You can share the report with other users - either by using "Save" button, or by base64-encoding the report query into a URL. You can also provide an optional title by adding `title` parameter to the URL, like this:
 
 ```console
-  HUB_URL="https://hub"
-  API="/index.php/advancedreports/#/report/run?sql="
-  SQL_QUERY="SELECT Hosts.HostName AS 'Host Name' FROM Hosts"
-  REPORT_TITLE="Example Report"
-  LINK="${HUB_URL}${API}$(echo ${SQL_QUERY} | base64)&title=$(/usr/bin/urlencode ${REPORT_TITLE})"
-  echo "${LINK}"
+HUB_URL="https://hub"
+API="/index.php/advancedreports/#/report/run?sql="
+SQL_QUERY="SELECT Hosts.HostName AS 'Host Name' FROM Hosts"
+REPORT_TITLE="Example Report"
+LINK="${HUB_URL}${API}$(echo ${SQL_QUERY} | base64)&title=$(/usr/bin/urlencode ${REPORT_TITLE})"
+echo "${LINK}"
```

Comment:
code block cleanup looked ok, except for the one cheat-sheet case where I think the indention was intended.